src/animation_curses.cpp

```diff
@@ -1,4 +1,4 @@
-#if (!defined SDLTILES)
```

Comment:
No point per se. I had to go through all the defines anyway, so why not change the preprocessor usage to a more canonical style? If / when I revert some the non-windows stuff, I can revert these small changes too.